integrations/amazon_bedrock/tests/test_chat_generator.py

```diff
@@ -173,6 +173,18 @@ def test_prepare_body_with_custom_inference_params(self) -> None:
 
         assert body == expected_body
 
+    @pytest.mark.integration
+    def test_get_responses(self) -> None:
+        adapter = AnthropicClaudeChatAdapter(generation_kwargs={})
+        response_body = {"completion": "This is a single response."}
+        expected_response = "This is a single response."
+        response_message = adapter.get_responses(response_body)
+        # assert that the type of each item in the list is a ChatMessage
+        for message in response_message:
```

Comment:
@davidsbatista added tests to confirm the types

Reply:
👍🏽